src/routes/__layout.svelte

```diff
@@ -1,53 +1,80 @@
 <script context="module">
+  import { XMLParser } from 'fast-xml-parser'
+  const parser = new XMLParser()
+
   export async function load({ fetch }) {
-    const res = await fetch(`/jobs/jobs.json`);
-    const { jobs } = await res.json();
-    const teamRes = await fetch(`/data/team.json`);
-    const { team } = await teamRes.json();
+    let jobs = []
+    const res = await fetch('https://climate-policy-radar.jobs.personio.com/xml')
```

Comment:
Should this link work for me? I get an error 🤔

Reply:
Yep should be turned off now - I needed to test turning it off in Personio. It's the reason I wrapped the parser in a try catch below here because I realised if we do turn it off at any point we can't have the website crash.